src/components/dao/settings-plan.vue

```diff
@@ -302,6 +317,10 @@ export default {
                   rounded
                   unelevated
                 )
+  widget(title="Billing history").full-width.q-mt-md
+    .calendar-container.q-mt-lg.row.q-gutter-sm
+      template(v-for="(bill, index) in BILLING_HISTORY")
+        billing-history-card(:planName="bill.planName" :status="bill.status" :date="new Date(bill.date)" :key="bill.id")
```

Comment:
@pugakn maybe you can clean this up with `v-bind="bill"` as prop name and billing vars are same name ? you just new to add to `BILLING_HISTORY` `new Date(bill.date)`